packages/react-components/react-menu/src/contexts/menuContext.ts

```diff
@@ -4,8 +4,11 @@ import type { ContextSelector, Context } from '@fluentui/react-context-selector'
 import type { MenuListProps } from '../components/index';
 import type { MenuState } from '../components/Menu/index';
 
-// eslint-disable-next-line @fluentui/no-context-default-value
-export const MenuContext: Context<MenuContextValue> = createContext<MenuContextValue>({
+export const MenuContext: Context<MenuContextValue | undefined> = createContext<MenuContextValue | undefined>(
+  undefined,
+);
+
+const menuContextDefaultValue = {
```

Comment:
```suggestion
const menuContextDefaultValue: MenuContextValue = {
```

nit: I would explicitly type it